src/cpp/DQM/serverext.cc

```diff
@@ -2384,7 +2385,15 @@ class VisDQMToJSON
 
       {
 	PyReleaseInterpreterLock nogil;
-	src->json(sample, path, options.count("rootcontent") > 0, options.count("lumisect") > 0, result, stamp, "", dirs);
+
+  // Create regex for the ME name
```

Comment:
Please fix the indentation.
No tabs, just spaces.

Reply:
Hi @rovere, actually I was thinking to auto format this entire file because it is very inconsistent. I will do it as a separate PR I think.

Reply:
Hi @rovere, if you don't have any objections, I will merge this and make a separate formatting PR.

Reply:
@andrius-k as you prefer. 
I'd have fixed this and then code-formatted the full file.